packages/runtime/container-runtime/src/summarizerTypes.ts

```diff
@@ -56,6 +56,7 @@ export interface ISummarizerInternalsProvider {
     /** Callback whenever a new SummaryAck is received, to update internal tracking state */
     refreshLatestSummaryAck(
         proposalHandle: string,
+        summaryRefSeq: number,
```

Comment:
nit: Consider adding this after the `ackHandle` param so the two handles are together.